opentech/apply/activity/messaging.py

```diff
@@ -111,19 +121,19 @@ class ActivityAdapter(AdapterBase):
         MESSAGES.NEW_SUBMISSION: 'Submitted {submission.title} for {submission.page.title}',
         MESSAGES.EDIT: 'Edited',
         MESSAGES.APPLICANT_EDIT: 'Edited',
-        MESSAGES.UPDATE_LEAD: 'Lead changed from {old.lead} to {submission.lead}',
-        MESSAGES.DETERMINATION_OUTCOME: 'Sent a determination. Outcome: {submission.determination.clean_outcome}',
+        MESSAGES.UPDATE_LEAD: 'Lead changed from {old_lead} to {submission.lead}',
+        MESSAGES.DETERMINATION_OUTCOME: 'Sent a determination. Outcome: {determination.clean_outcome}',
         MESSAGES.INVITED_TO_PROPOSAL: 'Invited to submit a proposal',
         MESSAGES.REVIEWERS_UPDATED: 'reviewers_updated',
-        MESSAGES.NEW_REVIEW: 'Submitted a review',
+        MESSAGES.NEW_REVIEW: MESSAGES.NEW_REVIEW.name,
```

Comment:
This prints out "NEW_REVIEW" instead of a fitting text.

Reply:
ah, I forgot to revert that change. Was looking at making the text that changes over time but decided against that as the number of related pages fetched would have been horrendous to make this happen.

Please revert